modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/NodeWithConsistencyToken.java

```diff
@@ -85,6 +84,6 @@ public int hashCode() {
     /** {@inheritDoc} */
     @Override
     public String toString() {
-        return S.toString(NodeWithConsistencyToken.class, this);
+        return name + ":" + enlistmentConsistencyToken;
```

Comment:
More concise representation, because  `nodeName: 123456789` is much more readable than `[NodeWithConsistencyToken [name=nodeName, enlistmentConsistencyToken=123456789] ]`.

Reply:
see comments above